MyWorkout.Web/Controllers/ExercisesController.cs

```diff
@@ -90,7 +88,7 @@ await _unitOfWork.ExerciseRepository
 
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction(nameof(ExercisesController.Details), "WorkoutDays", new { ID = exercise.WorkoutDayId });
```

Comment:
new { Id = exercise.WorkoutDayId } according to camel Case rule

Reply:
Updated